src/Microsoft.Azure.NotificationHubs/BaiduCredential.cs

```diff
@@ -4,11 +4,12 @@
 // license information.
 //------------------------------------------------------------
 
+using System;
+using Microsoft.Azure.NotificationHubs.Messaging;
```

Comment:
Please clean up the imports by using sort/remove using statements so that they're in order.

Reply:
Done